tfsdk/serve_test.go

```diff
@@ -3088,8 +3419,6 @@ func TestServerApplyResourceChange(t *testing.T) {
 			}
 			if tc.config.Type() != nil {
 				if diff := cmp.Diff(s.applyResourceChangeConfigValue, tc.config); diff != "" {
-					t.Errorf("Unexpected diff in config (+wanted, -got): %s", diff)
```

Comment:
This slightly concerns me; are we just not using this `if` check anymore?

Reply:
Wow, I have no idea where this completely unrelated change came from. Thanks for spotting it - reverted